index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel='stylesheet' href='styles.css'>
+    <title>Document</title>
```

Comment:
You should add a descriptive title to your page, it is an important element of an optimized SEO page.

Reply:
> You should add a descriptive title to your page, it is an important element of an optimized SEO page.

kindly adjust your phrases, the comments should sound more like suggestions instead of orders. For example, _"I think it would be better to ..."._